crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -594,6 +615,35 @@ module deepbook::clob_v2 {
         (ret_base_coin, ret_quote_coin, ret_val - original_val)
     }
 
+    // for smart routing
+    public fun swap_exact_base_for_quote_with_metadata<BaseAsset, QuoteAsset>(
+        pool: &mut Pool<BaseAsset, QuoteAsset>,
+        client_order_id: u64,
+        account_cap: &AccountCap,
```

Comment:
swap_exact_base_for_quote_with_metadata is a function based on original function swap_exact_base_for_quote. Hence, param ordering is followed as per swap_exact_base_for_quote function instead of place_market_order. Lemme know if you see any inconsistencies with this.